server/pulp/server/db/model/__init__.py

```diff
@@ -534,3 +534,24 @@ def __hash__(self):
         type and the same unit key will get the same hash value.
         """
         return hash(self.unit_type_id + self.unit_key_str)
+
+
+class CeleryBeatLock(Document):
+    """
+    Single document collection which gives information about the current celerybeat lock.
+
+    :param celerybeat_name: string representing the celerybeat instance name
```

Comment:
Very minor, but these are all instance variables, not params.

`:ivar celerybeat_name: string representing the celerybeat instance name`